ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -461,8 +480,9 @@ - (void)mapView:(MGLMapView *)mapView didFailToLocateUserWithError:(NSError *)er
 
 -(void)mapView:(MGLMapView *)mapView didSelectAnnotation:(id<MGLAnnotation>)annotation
 {
-    if (!annotation.title || !annotation.subtitle) { return; }
+    if (!annotation.title || !annotation.subtitle) { _selectedAnnotation=nil; return; }
```

Comment:
Split this line, please. This is only acceptable if you have a single statement in the block.